kytos/core/link.py

```diff
@@ -145,7 +145,8 @@ def get_next_available_tag(
         controller,
         link_id: str,
         take_last: bool = False,
-        tag_type: str = 'vlan'
+        tag_type: str = 'vlan',
+        avoid_tag: TAG = None,
```

Comment:
Aldo, the `TAG` class encapsulates both `tag_type: str` and its `value: int`, and since we had decided to support passing the `tag_type` explicitly in the allocation,   let's not give room here for `TAG` to have a different `tag_type`, so let's only pass its integer value `avoid_value: int`, see what I mean? Just so we keep it a bit more coherent by only accepting what's really necessary.

Reply:
Speaking of the `avoid_tag` and suggested `avoid_value: int`, can we rename it as `try_avoid_value: int`, meaning since your implementation just tries to avoid but can still return it it's not a hard avoid, by naming with a try in the var name it already gives the hind that it'll only try but not guaranteed, and then we can complement in the docstring explaining. Does that make sense?

Reply:
Changed `avoid_tag` to `try_avoid_value`. Modified docstring to explain `take_last` and `try_avoid_value`.